<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>529</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/projectIcon.ico</normaloff>:/images/projectIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:rgb(5,1,68);
}</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#f0f0ea;
	font-weight:bold;
	border: 1px solid black;
	border-radius:15px;
	color: black;
}

QPushButton:hover{
	background-color:#d8d8d3;
}</string>
   </property>
   <property name="text">
    <string>Organize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#f0f0ea;
	font-weight:bold;
	border-radius:15px;
	color:black;
}

QPushButton:hover{
	background-color:#d8d8d3;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This line input will be the target folder for the file organizing process.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:none;
	border-radius:10px;
	color:black;
	background-color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>C:/MyFolder/....</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Working Directory:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dirSlct">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid black;
	border-radius:10px;
	background-color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/ui/images/folder-tree-solid-full.svg</normaloff>:/images/ui/images/folder-tree-solid-full.svg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="excludeLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Exclude file formats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-image: url(:/button-bgs/ui/images/button-bgs/category1.png);
	border:none;
	border-radius:10px;
	color:black;
}

QPushButton:hover, QPushButton:checked{
	background-image:url(:/button-bgs/ui/images/button-bgs/category1-hover.png)
}
</string>
   </property>
   <property name="text">
    <string>Executables</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-image: url(:/button-bgs/ui/images/button-bgs/category2.png);
	border:none;
	border-radius:10px;
	color:black;
}

QPushButton:hover{
	background-image: url(:/button-bgs/ui/images/button-bgs/category2-hover.png);
}

QPushButton:hover, QPushButton:checked{
	background-image:url(:/button-bgs/ui/images/button-bgs/category2-hover.png)
}</string>
   </property>
   <property name="text">
    <string>Documents</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	background-image: url(:/button-bgs/ui/images/button-bgs/category3.png);
	border:none;
	border-radius:10px;
}

QPushButton:hover{
	background-image: url(:/button-bgs/ui/images/button-bgs/category3-hover.png);
}

QPushButton:hover, QPushButton:checked{
	background-image:url(:/button-bgs/ui/images/button-bgs/category3-hover.png)
}</string>
   </property>
   <property name="text">
    <string>Records</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	border:none;
	border-radius:10px;
	background-image: url(:/button-bgs/ui/images/button-bgs/category4.png);
}

QPushButton:hover, QPushButton:checked{
	background-image: url(:/button-bgs/ui/images/button-bgs/category4-hover.png);
}</string>
   </property>
   <property name="text">
    <string>Images</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	border:none;
	border-radius:10px;
	background-image: url(:/button-bgs/ui/images/button-bgs/category5.png);
}

QPushButton:hover, QPushButton:checked{
	background-image: url(:/button-bgs/ui/images/button-bgs/category5-hover.png);
}</string>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	border:none;
	border-radius:10px;
	background-image: url(:/button-bgs/ui/images/button-bgs/category6.png);
}

QPushButton:hover, QPushButton:checked{
	background-image: url(:/button-bgs/ui/images/button-bgs/category6-hover.png);
}</string>
   </property>
   <property name="text">
    <string>Videos</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	border:none;
	border-radius:10px;
	background-image: url(:/button-bgs/ui/images/button-bgs/category7.png);
}

QPushButton:hover, QPushButton:checked{
	background-image: url(:/button-bgs/ui/images/button-bgs/category7-hover.png);
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Archives</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fmtc_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:black;
	border:none;
	border-radius:10px;
	background-image: url(:/button-bgs/ui/images/button-bgs/category8.png);
}

QPushButton:hover{
	background-image: url(:/button-bgs/ui/images/button-bgs/category8-hover.png);
}

QPushButton:checked{
	background-image: url(:/button-bgs/ui/images/button-bgs/category8-checked.png);
}
</string>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Output Folder Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This line input will be the target folder for the file organizing process.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:none;
	border-radius:10px;
	color:black;
	background-color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>My Organized Files</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="skipMsg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Skip Popup Message</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn1</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
